File: soap_client.py
from suds.client import Client

client = Client('http://localhost:8000/?wsdl', cache=None)

print(client.service.say_hello(u'Maarika', 5))

wsdl = Client('http://www.learnwebservices.com/services/hello?wsdl', cache=None)
request = {'Name': 'Maarika'}
print(wsdl.service.SayHello(request))

File: soap_client.php
<?php

error_reporting(E_ALL);
ini_set('display_errors', 'On');
header('Content-Type: application/json');

$client = new SoapClient('http://localhost:8000/?WSDL');

$response = $client->say_hello(array('name' => 'Peeter', 'times' => '3'));

echo json_encode($response, JSON_PRETTY_PRINT);

?>